petastorm/pytorch.py

```diff
@@ -97,7 +96,8 @@ class DataLoader(object):
     runs out of samples.
     """
 
-    def __init__(self, reader, batch_size=1, collate_fn=decimal_friendly_collate):
+    def __init__(self, reader, batch_size=1, collate_fn=decimal_friendly_collate,
+                 shuffling_queue_capacity=0, min_after_dequeue=0):
```

Comment:
Add param descriptions for the new arguments

Reply:
Done. Added more documentation